app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -385,8 +431,9 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
       drawerLayout.post { drawerToggle.syncState() }
     } else {
       // For showing navigation drawer in AdministratorControlsActivity
```

Comment:
nit: add reference to developer options activity as well.

Reply:
Done. Added another comment below the if statement for DeveloperOptionsActivity on line 435.